addons/l10n_uy/demo/account_demo.py

```diff
@@ -1,20 +1,613 @@
+                'res.partner':
+                    self._get_demo_data_res_partner(company),
+            }
+        else:
+            return super(AccountChartTemplate)._get_demo_data(self,
+                                                              company=False)
+
+    def _post_load_demo_data(self, company=False):
+        if company.account_fiscal_country_id.code != "UY":
+            return super(AccountChartTemplate)._post_load_demo_data(company)
+        invoices = (
+            self.ref('demo_invoice_1')
+            + self.ref('demo_invoice_2')
+            + self.ref('demo_invoice_3')
+            + self.ref('demo_invoice_4')
+            + self.ref('demo_invoice_5')
+            + self.ref('demo_invoice_6')
+            + self.ref('demo_invoice_8')
+            + self.ref('demo_invoice_9')
+            + self.ref('demo_sup_invoice_1')
+            + self.ref('demo_sup_invoice_2')
+            + self.ref('demo_sup_invoice_3')
+            + self.ref('demo_sup_invoice_6')
+            + self.ref('demo_sup_invoice_7')
+            + self.ref('demo_sup_invoice_8')
+            + self.ref('demo_sup_invoice_9')
+        )
+        # the invoice_extract acts like a placeholder for the OCR to be ran and
+        # doesn't contain any lines yet
+        for move in invoices:
+            try:
+                move.action_post()
+            except (UserError, ValidationError):
+                _logger.exception('Error while posting invoices')
+
+        # Post the reversal moves
+        invoices_to_revert = (
+            self.ref('demo_refund_invoice_1')
+            + self.ref('demo_refund_invoice_2')
+            + self.ref('demo_refund_invoice_3')
+            + self.ref('demo_refund_invoice_4')
+            + self.ref('demo_sup_refund_invoice_3')
+            + self.ref('demo_sup_refund_invoice_2')
+            )
+        for move in invoices_to_revert:
+            try:
+                self.env['account.move'].browse(
+                    move.refund_moves().get('res_id')).action_post()
+            except (UserError, ValidationError):
+                _logger.exception('Error while posting reversal moves')
+
     @api.model
     def _get_demo_data_move(self, company=False):
-        """ Set the l10n_latam_document_number on demo invoices """
-        move_data = super()._get_demo_data_move(company)
-        if company.account_fiscal_country_id.code == 'UY':
-            number = 1
-            for move in move_data.values():
-                if move['move_type'] in ('in_invoice', 'in_refund'):
-                    move['l10n_latam_document_number'] = 'A' + f'{number:07d}'
-                    number += 1
-
-        return move_data
+        cid = company.id or self.env.company.id
+        sale_journal = self.env['account.journal'].search(
+            domain=[
+                *self.env['account.journal']._check_company_domain(cid),
+                ('type', '=', 'sale')
+            ], limit=1)
```

Comment:
```suggestion
            ], limit=1,
        )
```
Same for the other calls like it